.golangci.yml

```diff
@@ -9,7 +9,6 @@ linters:
   enable:
     - asciicheck
     - bidichk
-    - depguard
```

Comment:
Don't know if it's just for me, but my golang linter started going crazy with dep-guard (import 'github.com/cosmos/cosmos-sdk/store/types' is not allowed from list 'Main' (depguard)go-golangci-lint)

Reply:
Tbh would prefer if we get to the bottom of the issue / discuss this in a separate PR to avoid blocking this.

Not having any issues myself. Thinking that if we disable a linter every time someone has a local error, the linter will stop being valuable!

Reply:
no worries, if dep guard was my local issue, happy to put it back in